### PR TITLE
Fix dotcom Welcome Guide to be safe for atomic.

### PR DESCRIPTION
**Pull request by Addison-Stavlo**
#### Changes proposed in this Pull Request

A potential solution for bringing this back into atomic.  Essentially we only register the nux plugin if the experimental function is available.  This is the most recent addition to the dependencies in this file so if it is available, everything should be.  Similarly, if this function name gets changed as it gets determined to be a full feature and not an experiment, we will have a back up check to prevent this from breaking on simple sites as well.

#### Testing instructions

* Run this FSE build on Atomic development sites.
* Test loading the editor with and without gutenberg enabled.

Fixes #

### CODE REVIEW

File: apps/full-site-editing/full-site-editing-plugin/wpcom-block-editor-nux/src/wpcom-nux.js [hunk 1]
```diff
@@ -111,6 +111,8 @@ function WpcomNux() {
 	);
 }
 
-registerPlugin( 'wpcom-block-editor-nux', {
-	render: () => <WpcomNux />,
-} );
+if ( __experimentalCreateInterpolateElement && Guide && GuidePage ) {
```
Comment (vindl) [inline apps/full-site-editing/full-site-editing-plugin/wpcom-block-editor-nux/src/wpcom-nux.js]: I think we should add a comment explaining why this is needed.

Comment (Addison-Stavlo) [reply apps/full-site-editing/full-site-editing-plugin/wpcom-block-editor-nux/src/wpcom-nux.js]: added

**Comment by vindl** [comment]
> When I have the FSE plugin activated and the Gutenberg plugin deactivated, I do not see any NUX.
> I think this is expected behavior?

With the existing code this was falling back to the old block editor NUX. It might be fine not to show it since it will be very different from the Dotcom one.

**Comment by vindl** [comment]
> However, I don't think that the setting for showing the guide is syncing between wpcom and my atomic site. do we expect it to? maybe it has something to do with how atomic dev sites work.

I'm not sure that the ephemeral Atomic site is created for the same user ID, and then it would be stored in a different user preference.

**Comment by Addison-Stavlo** [comment]
> When I have the FSE plugin activated and the Gutenberg plugin deactivated, I do not see any NUX.
I think this is expected behavior?

I don't think that old version even has a welcome guide yet, its definitely not available in the menu.  If it did, we are effectively disabling it which shouldn't be an issue considering we were disabling the guide entirely until we started this re-enabling adventure in December.

**Comment by vindl** [comment]
> I don't think that old version even has a welcome guide yet, its definitely not available in the menu.

It's not available in the menu but it can pop up if you clear the storage. It's the old one with notes popping around the screen for various UI items. 😄

**Comment by Addison-Stavlo** [comment]
> It's not available in the menu but it can show up if you clear the storage. It's the old one with notes popping around the screen for various UI items. 😄

Gotcha, so we will be hiding that with this:
https://github.com/Automattic/wp-calypso/blob/bc3194d1e436751c6dd106e93ad53e40cbc42dda/apps/full-site-editing/full-site-editing-plugin/wpcom-block-editor-nux/src/disable-core-nux.js#L9-L15

But given that we ended up hiding that nux version on dotcom when it was a thing, it sounds reasonable here.

**Comment by vindl** [comment]
> But given that we ended up hiding that nux version on dotcom when it was a thing, it sounds reasonable here.

I agree. 👍

**Pull request merged**